Update night.json

lexxanite opened the pull request:
This is an update to reflect the differences between the app version with the physical version of Blood on the Clocktower.

The original branch was based on the app, but the tool is meant to create physical night order sheets. To reflect the properties of a physical version, the following changes have been made:

-The usage of "and" has been changed to "&" 
---This brings it in line with the printed versions of Trouble Brewing, Bad Moon Rising, and Sects and Violets.

-Dusk has been changed from "Start the night." to "Check that all eyes are closed. Some Fabled act."
---This brings it in line with the printed versions of Trouble Brewing, Bad Moon Rising, and Sects and Violets. Small liberties have been made, as the tool is adding a traveler night order reminder, and the physical versions say "Some Travellers & Fabled act." It's a design decision to remove "Travellers &" to not be redundant with the next line in the night order.

-Dawn has been changed from "Wait for a few seconds. End the Night Phase." to "Wait for a few seconds. Call for eyes open."
---This brings it in line with the printed versions of Trouble Brewing, Bad Moon Rising, and Sects and Violets.

lexxanite committed:
Update night.json

djad99 commented on night.json:
```diff
@@ -332,7 +332,7 @@
             },
             {
-                "description": "The Engineer might choose Minions or Demons. \u26ab If they do: \n\tPut the Engineer to sleep. Wake a targer, show them the YOU ARE token and their new character token, then put that target to sleep. Repeat for all players that changed characters."
+                "description": "The Engineer might choose Minions or Demons. \u26ab If they do: \n\tPut the Engineer to sleep. Wake a targer, show them the YOU ARE token & their new character token, then put that target to sleep. Repeat for all players that changed characters."
```
Oops on this line from my code if you could fix it

"Wake a targer" instead of "Wake a target"

djad99 commented on night.json:
```diff
@@ -1188,7 +1188,7 @@
             },
             {
-                "description": "The Engineer might choose Minions or Demons. \u26ab If they do: \n\tPut the Engineer to sleep. Wake a targer, show them the YOU ARE token and their new character token, then put that target to sleep. Repeat for all players that changed characters."
+                "description": "The Engineer might choose Minions or Demons. \u26ab If they do: \n\tPut the Engineer to sleep. Wake a targer, show them the YOU ARE token & their new character token, then put that target to sleep. Repeat for all players that changed characters."
```
Repeat comment from the previous one

"Wake the targer" instead of "Wake the target"

djad99 commented on night.json:
```diff
@@ -1608,7 +1608,7 @@
             },
             {
-                "description": "The Al-Hadikhia chooses three players. \u26ab\u26ab\u26ab Wake the player marked 1 and say 'The Al-Hadikhia has chosen', then the player's name, then 'Do you choose to live?' They either nod or shake their head. Put them to sleep and add or remove shrouds accordingly. Repeat for players marked 2 and 3. If all three players are now alive, add a shroud to all three."
+                "description": "The Al-Hadikhia chooses three players. \u26ab\u26ab\u26ab Wake the player marked 1 & say 'The Al-Hadikhia has chosen', then the player's name, then 'Do you choose to live?' They either nod or shake their head. Put them to sleep & add or remove shrouds accordingly. Repeat for players marked 2 & 3. If all three players are now alive, add a shroud to all three."
```
three -> 3

lexxanite committed:
Update night.json

djad99 commented on night.json:
```diff
@@ -1608,7 +1608,7 @@
             },
             {
-                "description": "The Al-Hadikhia chooses three players. \u26ab\u26ab\u26ab Wake the player marked 1 & say 'The Al-Hadikhia has chosen', then the player's name, then 'Do you choose to live?' They either nod or shake their head. Put them to sleep & add or remove shrouds accordingly. Repeat for players marked 2 & 3. If all three players are now alive, add a shroud to all three."
+                "description": "The Al-Hadikhia chooses 3 players. \u26ab\u26ab\u26ab Wake the player marked 1 & say 'The Al-Hadikhia has chosen', then the player's name, then 'Do you choose to live?' They either nod or shake their head. Put them to sleep & add or remove shrouds accordingly. Repeat for players marked 2 & 3. If all three players are now alive, add a shroud to all three."
```
"If all 3 players are now alive, add a shroud to all 3"

lexxanite committed:
Update night.json

Correcting Al-Had text per https://github.com/djad99/Blood-on-the-Clocktower-Night-Order-Instructions/pull/1#discussion_r1950220792
